### PR TITLE
Sidebar and Bottombar navigation

### PR DESCRIPTION
**Pull request by JayVnl**

### CODE REVIEW

**Review by DannyPX — APPROVED** [review]

File: app/src/main/java/com/djinc/edumotive/MainActivity.kt [hunk 1]
```diff
@@ -3,41 +3,51 @@ package com.djinc.edumotive
...
+            val windowSize = rememberWindowSizeClass()
             EdumotiveTheme {
-                // A surface container using the 'background' color from the theme
                 Surface(
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                    Greeting("Android")
+                    Main(windowSize)
+//                    Row {
```
Comment (DannyPX) [inline app/src/main/java/com/djinc/edumotive/MainActivity.kt:39]: Remove comments?

File: app/src/main/java/com/djinc/edumotive/components/SideBar.kt [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                    modifier = Modifier
+                        .width(20.dp)
+                        .height(20.dp)
+                        .background(Color.Black)
+                )
+                Spacer(modifier = Modifier.width(10.dp))
+                Text("EduMotive")
+            }
+            Spacer(modifier = Modifier.height(24.dp))
+            Row(modifier = Modifier.clickable {
```
Comment (DannyPX) [inline app/src/main/java/com/djinc/edumotive/components/SideBar.kt:38]: Repetitive coding for each item. Make a list and loop through it.

File: app/src/main/java/com/djinc/edumotive/screens/Main.kt [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.navigation.compose.rememberNavController
+import com.djinc.edumotive.components.BottomBar
+import com.djinc.edumotive.components.SideBar
+import com.djinc.edumotive.navigation.NavGraph
+import com.djinc.edumotive.utils.WindowSize
+
+@Composable
+fun Main(windowSize: WindowSize) {
```
Comment (DannyPX) [inline app/src/main/java/com/djinc/edumotive/screens/Main.kt:18]: Rename it to App..., for standardisation

**Review by DannyPX — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/djinc/edumotive/MainActivity.kt (hunk 1 above)
Comment (JayVnl) [reply app/src/main/java/com/djinc/edumotive/MainActivity.kt]: Code is indeed deprecated and has been removed

**Review by DannyPX — APPROVED** [review]

**Pull request merged**